src/fsharp/FSharp.Core/string.fsi

```diff
@@ -137,3 +138,165 @@ namespace Microsoft.FSharp.Core
+        /// <param name="str">The input string.</param>
+        /// 
+        /// <returns>
+        ///     A string that is equivalent to this instance except that all occurences of <c>oldValue</c> are replaced
+        ///     with <c>newValue</c>. If <c>oldValue</c> is not found in <c>str</c>, the function returns <c>str</c>
+        ///     unchanged.
+        /// </returns>
+        /// <exception cref="T:System.ArgumentNullException">Thrown when <c>str</c> is null.</exception>
+        [<CompiledName("Replace")>]
+        val replace: oldValue:string -> newValue:string -> str:string -> string
+
+        /// <summary>
+        ///     Returns a new string in which all occurrences of a specified Unicode character in an input string are
+        /// replaced with another Unicode character.
+        /// </summary>
+        ///
+        /// <param name="oldChar">The Unicode character to be replaced.</param>
+        /// <param name="newChar">The Unicode character to replace all occurences of <c>oldChar</c>.</param>
+        /// <param name="str">The input string.</param>
+        /// 
+        /// <returns>
+        ///     A string that is equivalent to this instance except that all occurences of <c>oldChar</c> are replaced
+        ///     with <c>newChar</c>. If <c>oldChar</c> is not found in <c>str</c>, the function returns <c>str</c>
+        ///     unchanged.
+        /// </returns>
+        /// <exception cref="T:System.ArgumentNullException">Thrown when <c>str</c> is null.</exception>
+        [<CompiledName("ReplaceChar")>]
+        val replaceChar: oldChar:char -> newChar:char -> str:string -> string
+        
+        /// <summary>
+        ///     Returns a new string in which all leading and trailing white-space characters in an input string are
+        ///     removed. White-space characters are defined by any character for which <c>System.Char.IsWhiteSpace</c>
+        ///     returns <c>true</c>.
+        /// </summary>
+        ///
+        /// <param name="str">The input string.</param>
+        /// 
+        /// <returns>The resulting string.</returns>
+        /// <exception cref="T:System.ArgumentNullException">Thrown when <c>str</c> is null.</exception>
+        [<CompiledName("Trim")>]
+        val trim: str:string -> string
+        
+        /// <summary>
+        ///     Returns a new string in which all leading and trailing occurences of a set of specified characters 
+        ///     in an input string are removed.
+        /// </summary>
+        ///
+        /// <param name="chars">The set of strings to remove.</param>
+        /// <param name="str">The input string.</param>
+        /// 
+        /// <returns>The resulting string.</returns>
+        /// <exception cref="T:System.ArgumentNullException">Thrown when <c>str</c> is null.</exception>
+        [<CompiledName("TrimChars")>]
+        val trimChars: chars:seq<char> -> str:string -> string
```

Comment:
The problem with this sort of thing is it's really not an improvement on `s.Trim('a')` with ParamArrays, it brings nothing but pipelining.  That's not a sufficient reason to be in FSharp.Core